dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -472,5 +472,9 @@ variables.in.id.not.supported=CWWKG0104W: The ID attribute with value {1} for th
 variables.in.id.not.supported.explanation: Variable values are not allowed in ID attributes. All variables in ID attributes will be treated as literal strings.  
 variables.in.id.not.supported.useraction: Remove the variable from the value of the ID attribute.
 
+error.root.must.be.server=CWWKG0105E: The configuration document {0} must begin with the element '{1}'.
+error.root.must.be.server.explanation=Configuration must be contained in either a server element or a client element. 
```

Comment:
how will the user know whether a client or server element is required?

Reply:
It's in the error message ( the {1} substitution )

Reply:
ok- if server or client is directly specified in the variable I think this is ok. Add "the"

The configuration must be contained in either a server element or a client element.